src/VisualStudio/Core/Def/Commands.vsct

```diff
@@ -489,19 +413,9 @@
     <GuidSymbol name="guidVisualBasicCodePageEditorFactory" value="{6c33e1aa-1401-4536-ab67-0e21e6e569da}" />
 
     <GuidSymbol name ="guidCSharpGrpId" value ="{5d7e7f65-a63f-46ee-84f1-990b2cab23f9}">
-      <IDSymbol name="IDM_CSHARP_ORGANIZE" value="0x3900"/>
-      <IDSymbol name="IDM_CSHARP_CTX_ORGANIZE" value="0x3950"/>
-
       <IDSymbol name="IDG_CSHARP_CODEGEN" value="0x3490"/>
-      <IDSymbol name="IDG_CSHARP_ORGANIZE" value="0x3617"/>
-      <IDSymbol name="IDG_CSHARP_CTX_ORGANIZE" value="0x3618"/>
 
-      <IDSymbol name="cmdidContextOrganizeRemoveUnusedUsings" value="0x1911"/>
```

Comment:
I know I've seen extensions for VS in the past that would explicitly call through with these commands to programmatically do that. That was before we had a "fix all in solution" so it's possible we don't care, but you might want to do a quick search through the VS codebase to make sure nobody is relying on us still responding to these.